.github/CODEOWNERS

```diff
@@ -307,12 +337,18 @@
 # PRLabel: %EngSys
 /sdk/template/                                                       @scbedd @weshaggard @benbp
 
+# AzureSDKOwners: @kristapratico
```

Comment:
Does our team still own TA?  I thought we were transferring ownership to the service team across languages.   If so, we'll need to remove AzureSDKOwners here so that the service owners defined below are considered the issue owners.